### PR TITLE
Move from suite.suite to testing.T

### PR DESCRIPTION
**Pull request by sethiay**
### Description
Move from suite.suite to testing.T as per best practices of writing tests.

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - NA
2. Unit tests - NA
3. Integration tests - NA

### CODE REVIEW

File: internal/cache/file/cache_handler_test.go [hunk 1]
```diff
@@ -523,112 +562,117 @@ func (chrT *cacheHandlerTest) Test_GetCacheHandle_ConcurrentDifferentFiles() {
...
-	require.True(chrT.T(), chrT.isEntryInFileInfoCache(chrT.object.Name, chrT.bucket.Name()))
+func Test_InvalidateCache_ConcurrentSameFile(t *testing.T) {
+	cacheDir := path.Join(os.Getenv("HOME"), "CacheHandlerTest/dir")
+	chTestArgs := initializeCacheHandlerTestArgs(t, &config.FileCacheConfig{EnableCRC: true}, cacheDir)
+	existingJob := getDownloadJobForTestObject(t, chTestArgs)
+	require.Equal(t, downloader.NotStarted, existingJob.GetStatus().Name)
+	require.True(t, isEntryInFileInfoCache(t, chTestArgs.cache, chTestArgs.object.Name, chTestArgs.bucket.Name()))
 	wg := sync.WaitGroup{}
-	InvalidateCacheTestFun := func() {
+	InvalidateCacheTestFun := func(t *testing.T) {
```
Comment (kislaykishore) [inline internal/cache/file/cache_handler_test.go:640]: This being an inner function should already have access to "t" from the outer scope. So, we probably don't need this parameter.

**Review by kislaykishore — CHANGES_REQUESTED** [review]

File: internal/cache/file/cache_handler_test.go [hunk 2]
```diff
@@ -522,89 +562,92 @@ func (chrT *cacheHandlerTest) Test_GetCacheHandle_ConcurrentDifferentFiles() {
...
+	require.False(t, cacheHit)
+	require.Nil(t, cacheHandle.Close())
 	// Open cache file before invalidation
-	objectPath := util.GetObjectPath(chrT.bucket.Name(), minObject.Name)
-	downloadPath := util.GetDownloadPath(chrT.cacheDir, objectPath)
+	objectPath := util.GetObjectPath(chTestArgs.bucket.Name(), minObject.Name)
+	downloadPath := util.GetDownloadPath(chTestArgs.cacheDir, objectPath)
 	file, err := os.OpenFile(downloadPath, os.O_RDONLY, 0600)
-	require.NoError(chrT.T(), err)
-	_, err = file.Read(buf)
```
Comment (sethiay) [inline internal/cache/file/cache_handler_test.go:579]: this is intended, we can skip this assertion.

File: internal/cache/file/cache_handler_test.go [hunk 3]
```diff
@@ -522,89 +562,92 @@ func (chrT *cacheHandlerTest) Test_GetCacheHandle_ConcurrentDifferentFiles() {
...
+	require.NoError(t, err)
+	require.Equal(t, string(objectContent[:3]), string(buf))
+	require.False(t, cacheHit)
+	require.Nil(t, cacheHandle.Close())
 	// Open cache file before invalidation
-	objectPath := util.GetObjectPath(chrT.bucket.Name(), minObject.Name)
-	downloadPath := util.GetDownloadPath(chrT.cacheDir, objectPath)
+	objectPath := util.GetObjectPath(chTestArgs.bucket.Name(), minObject.Name)
+	downloadPath := util.GetDownloadPath(chTestArgs.cacheDir, objectPath)
 	file, err := os.OpenFile(downloadPath, os.O_RDONLY, 0600)
```
Comment (kislaykishore) [inline internal/cache/file/cache_handler_test.go:622]: Should we close the file in the test?

**Review by kislaykishore — APPROVED** [review]

**Review by kislaykishore — APPROVED** [review]

**Review by ashmeenkaur — APPROVED** [review]
Relying on @kislaykishore's review

**Pull request merged**